Apps/Sandcastle/gallery/CZML Rectangle.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+  <meta name="description" content="CZML Rectangle">
+  <meta name="cesium-sandcastle-labels" content="CZML">
+  <title>Cesium Demo</title>
+  <script type="text/javascript" src="../Sandcastle-header.js"></script>
+  <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+  <script type="text/javascript">
+  require.config({
+    baseUrl : '../../../Source',
+    waitSeconds : 60
+  });
+  </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+@import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+  "use strict";
+  //Sandcastle_Begin
+  var czml = [
+    {
+      "id" : "document",
+      "name" : "CZML Geometries: Box",
+      "version" : "1.0"
+    }, {
+      "id" : "redRectangle",
+      "rectangle" : {
+        "show" : true,
+        "coordinates" : { "wsenDegrees" : [-120,40,-110,50] },
+        "height" : 600000,
+        "extrudedHeight" : 0,
+        "fill" : true,
+        "material" : {
+          "solidColor" : {
+            "color": {
+              "rgba" : [255,0,0,100]
+            }
+          }
+        },
+        "outline" : true,
+        "outlineColor" : {
+          "rgbaf" : [0, 0, 0, 1.0]
+        }
+      }
+    }, {
+      "id" : "blueRectangle",
+      "rectangle" : {
+        "show" : true,
+        "coordinates" : { "wsenDegrees" : [-105,40,-95,50] },
+        "height" : 0,
+        "fill" : true,
+        "material" : {
+          "stripe" : {
+            "orientation" : "VERTICAL",
+            "evenColor" : { "rgba" : [0, 255, 0, 255] },
+            "oddColor" : { "rgba" : [0, 0, 255, 255] },
+            "repeat" : 5
+          }
+        }
+      }
+    },
+    {
+      "id" : "greenRectangle",
```

Comment:
@pjcozzi I've noticed that if you don't explicitly define a `"name"`, objects seem to use the `"id"` as their display name? Or am I misunderstanding the purpose of the `"name"` property (as I'm assuming it's simply the below information)?
![image](https://cloud.githubusercontent.com/assets/6036502/10184470/076a8e06-670b-11e5-824f-37844db2a0d7.png)